fastlane/spec/actions_specs/push_git_tags_spec.rb

```diff
@@ -1,13 +1,37 @@
 describe Fastlane do
   describe Fastlane::FastFile do
     describe "push_git_tags" do
-      it "uses the correct comand" do
+      it "uses the correct default comand" do
         result = Fastlane::FastFile.new.parse("lane :test do
           push_git_tags
         end").runner.execute(:test)
 
         expect(result).to eq("git push --tags")
       end
+
+      it "uses the correct comand when forced" do
+        result = Fastlane::FastFile.new.parse("lane :test do
+          push_git_tags(force: true)
+        end").runner.execute(:test)
+
+        expect(result).to eq("git push --tags --force")
+      end
+
+      it "uses the correct comand when remote set" do
+        result = Fastlane::FastFile.new.parse("lane :test do
+          push_git_tags(remote: 'foo')
+        end").runner.execute(:test)
+
+        expect(result).to eq("git push --tags foo")
```

Comment:
Additionally if you look at the manual page for git

`SYNOPSIS
       git push [--all | --mirror | --tags] [--follow-tags] [--atomic] [-n | --dry-run] [--receive-pack=<git-receive-pack>]
                  [--repo=<repository>] [-f | --force] [--prune] [-v | --verbose]
                  [-u | --set-upstream]
                  [--[no-]signed|--sign=(true|false|if-asked)]
                  [--force-with-lease[=<refname>[:<expect>]]]
                  [--no-verify] [<repository> [<refspec>...]]`

The repository should go at the end.